hawc/apps/eco/forms.py

```diff
@@ -1,48 +1,147 @@
 from django import forms
 
-from ..common.admin import autocomplete
-from . import models
+from hawc.apps.common.forms import BaseFormHelper
+
+from ..common import selectable
+from . import lookups, models
 
 
 class DesignForm(forms.ModelForm):
+    CREATE_LEGEND = "Create new study design"
+    CREATE_HELP_TEXT = ""
+    UPDATE_HELP_TEXT = "Update an existing study design."
+
+    country = selectable.AutoCompleteSelectMultipleWidget(lookup_class=lookups.CountryLookup)
+    state = selectable.AutoCompleteSelectMultipleWidget(lookup_class=lookups.StateLookup)
+    ecoregion = selectable.AutoCompleteSelectMultipleWidget(lookup_class=lookups.EcoregionLookup)
+
     class Meta:
-        fields = "__all__"
+        exclude = ("study",)
         model = models.Design
-        widgets = {
-            "study": autocomplete(models.Design, "study"),
-            "countries": autocomplete(models.Design, "countries", multi=True),
-            "state": autocomplete(models.Design, "state", multi=True),
-            "ecoregion": autocomplete(models.Design, "ecoregion", multi=True),
-        }
+
+    def __init__(self, *args, **kwargs):
+        study = kwargs.pop("parent", None)
+        super().__init__(*args, **kwargs)
+        if study:
+            self.instance.study = study
+
+    @property
+    def helper(self):
+        for fld in ("habitat_as_reported", "climate_as_reported"):
+            self.fields[fld].widget.attrs["class"] = "html5text"
```

Comment:
remove the quill attributes from this; these can just be plain text, not quill editors